recipes/betse/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "betse" %}
+{% set version = "0.7.0" %}
+{% set file_ext = "tar.gz" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ file_ext }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  sha256: 845896f4c30b3970d5bd9501e0ca754eef5af0d742de76103f66c8c8c58e4eca
```

Comment:
set the `sha256` as a jinja variable at the top.

Reply:
Ah, right. One last edit for the evening, then. <sup>It's always "one last edit".</sup>